manifests/params.pp

```diff
@@ -73,6 +73,7 @@
   $config                      = 'rabbitmq/rabbitmq.config.erb'
   $config_cluster              = false
   $config_path                 = '/etc/rabbitmq/rabbitmq.config'
+  $config_ranch                = true
```

Comment:
How about ```$config_ranch = !$facts['rabbitmq_version'] || versioncmp($facts['rabbitmq_version'], '3.6') >= 0```

Reply:
will the `versioncmp()` call translate directly into a bool automagically?

Reply:
I like this idea in general, just wondering if we're trying to be too clever and might end up shooting ourself in the foot?
Also, maybe it would be easier to read if we just had an `if {}` block rather than keeping it so terse?

Reply:
> will the versioncmp() call translate directly into a bool automagically?

Maybe you need braces around the comparison, haven't tested it.